### DISCUSSION TITLE
stealth mode for cinematography

### DISCUSSION THREAD

**Discussion by Czukuru** [gopro/labs#146] [Q&A]
Hi. I need 2 qr codes for stealth using in the frame of big production camera. With one qr i want to turn off all led's and both screens with permanent command to turn off the camera before the motion and than hit remote rec button with no light from the camera. And after that, another qr to turn everything on

**Comment by Czukuru** [comment]
I know how to turn off red led's. But i can't find the command to turn off screens. Only brightness or turn off front screen from the camera, not from qr

**Comment by dnewman-gpsw** [comment]
QR command for turning off the front screen is oF0, and on again oF1.  There is on command for completely disabling the rear screen, likely to prevent users thinking they have bricked their camera.  The best you can do currently is oB1D0S1F0    <- Brightness 10%, no LEDs, screen off at 1minute, and front screen off.    The reverse is oB9D4SF1 (this places the screens always on.)

**Reply by Czukuru** [reply]
Thanks. You really helps me

**Comment by dnewman-gpsw** [comment]
Also another solution just discovered. The command oDO - option Display Off, will immediately turn off the LCD. Touching the screen will turn it back on again, so it is pretty save to use.  So I combined this with a shorting of the on-screen messaging and a BOOT command (in qrcode below) and you have the LCD on for about a second at maximum. 
![image](https://user-images.githubusercontent.com/10856084/205181870-81c7cb9e-aa84-4f95-b00a-93f6ad7fe31d.png)

https://user-images.githubusercontent.com/10856084/205182083-d2d094bc-1e00-41d2-b507-64c351e96dea.mp4
